staff/RominaFior/js/challenges/scripts/toRomanNumeral.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+function toRomanNumeral(num) {
+    
+    if (typeof num !== 'number' )
+        throw Error('input num is not a number');
+    
+    switch (num) {
+        
+    
+    case 1:
+        num = "I";
```

Comment:
use `return "I"` instead of `num = "I"`

Reply:
@alexgbor Why, you need only one return per funtion

Reply:
🤔 Hm.. true!